src/api/api.js

```diff
@@ -2,12 +2,16 @@ import axios from "axios";
 
 function fetchArticles(){
     return axios.get("https://nc-news-6ukh.onrender.com/api/articles")
-    .then((response)=>{
-        console.log(response.data, "<-- .then clause")
-        return response.data;
-    }).then((data)=>{
-        return data.articles;
+    .then((response) => {
+        return response.data.articles;
     })
 }
 
-export { fetchArticles };
\ No newline at end of file
+function fetchArticleById(articleId){
+    return axios.get(`https://nc-news-6ukh.onrender.com/api/articles/${articleId}`)
```

Comment:
Nice work using axios here. See if you can make your life `even easier` and implement a baseURL in this file too 👍